scipy/signal/tests/test_spectral.py

```diff
@@ -18,120 +17,200 @@
 from scipy.signal.tests._scipy_spectral_test_shim import stft_compare as stft
 from scipy.signal.tests._scipy_spectral_test_shim import istft_compare as istft
 from scipy.signal.tests._scipy_spectral_test_shim import csd_compare as csd
+from scipy.conftest import array_api_compatible, skip_if_array_api_backend
+from scipy._lib.array_api_compat import array_api_compat
```

Comment:
This import path was cleaned up recently.
```suggestion
from scipy._lib import array_api_compat
```